pandas/core/indexes/base.py

```diff
@@ -122,6 +122,24 @@ class Index(IndexOpsMixin, PandasObject):
     Notes
     -----
     An Index instance can **only** contain hashable objects
+
+    Examples
+    --------
+    >>> pd.Index([1, 2, 3])
+    Int64Index([1, 2, 3], dtype='int64')
+
+    >>> pd.Index(list('abc'))
+    Index(['a', 'b', 'c'], dtype='object')
+
+    See Also
+    ---------
+    RangeIndex : Index implementing a monotonic integer range (very
+                 memory-efficient)
+    CategoricalIndex : Index with :class:`Categorical` s.
```

Comment:
Nitpick: This description is essentially structured the same way as `IntervalIndex` below, but worded slightly differently, "Index with" vs. "an Index of".  This could be made consistent.